controllers/grid/users/author/form/PKPAuthorForm.inc.php

```diff
@@ -26,13 +26,19 @@ class PKPAuthorForm extends Form {
 	 * Constructor.
 	 */
 	function __construct($publication, $author) {
-		parent::__construct('controllers/grid/users/author/form/authorForm.tpl');
+		$submission = Services::get('submission')->get($publication->getData('submissionId'));
+		$context = Services::get('context')->get($submission->getData('contextId'));
```

Comment:
Both of these are expensive operations, because the objects are so large. We should try to pass them into `new AuthorForm()` from the objects that already exist in `AuthorGridHandler`:

```
$form = new AuthorForm(
    $this->getPublication(),
    $author,
    $this->getSubmission(),
    $request->getContext()
);
```

There are two places that `AuthorForm` is instantiated in the grid.

Reply:
Looks good!